tomviz/DataTransformMenu.cxx

```diff
@@ -38,6 +39,8 @@ void DataTransformMenu::buildTransforms()
 
   auto cropDataAction = menu->addAction("Crop");
   auto convertDataAction = menu->addAction("Convert to Float");
+  auto arrayWranglerAction =
+    menu->addAction("Wrangle Data");
```

Comment:
I think we should call it "Convert to Type" in the menu, the class names are fine, but it may not be obvious to end users. Not sure on that, but that is my second attempt at menu name.